build.sh

```diff
@@ -1,37 +1,77 @@
-set -e 
+#!/bin/sh
+# Build script of python package
+
+set -e
 name="oval-graph"
 module="oval_graph"
 
+unit_test() {
+    python3 -m pytest
+}
+
+commit_version() {
+    version=$1
+    module=$2
+    git add ${module}/__init__.py
+    git add oval-graph.spec
+    git commit -m "${version}"
+    git tag "${version}"
+    git push --follow-tags
+    git push upstream master
+}
+
+install_build_requirements() {
+    python3 -m pip install --user --upgrade setuptools wheel twine
+}
+
+build_backage() {
+    rm -rf dist
+    python3 setup.py sdist bdist_wheel
+    twine upload dist/*
+}
+
+update_version_package() {
+    module=$1
+    sed -i "s/$old_version/$new_version/g" ${module}/__init__.py
+}
+
+update_version_rpm() {
+    version=$1
+    name=$2
+    rpmdev-bumpspec "${name}".spec --comment="release ${version}" -n "${version}" --userstring="Jan Rodak <jrodak@redhat.com>"
+}
+
+load_upstream() {
+    git checkout master
+    git pull upstream master
+}
+
 if [ "$1" != "" ]; then
     new_version=$1
-    if [[ $new_version =~ ^[0-9]{1,}.[0-9]{1,}.[0-9]{1,}$ ]]; then
```

Comment:
Those nested `if`s are difficult to read. You can use `exit` or define and use a `die` function to simply stop the script if something is not OK. That way, you won't have to have an `else` block.